[ADD] Frontdesk: add new configuration doc

larm-odoo opened the pull request:
No documentation exists for this new app that was created in version 17.0. Adding new instructions for Frontdesk.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/7237).

jero-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+:guilabel:`Nothing, thanks` button at the bottom of the screen. If a drink selection was made, a
+`Thank you for registering! Your drink is on the way.` message appears.
+
+At any point in the check-in process, if ten (10) seconds passes with no selection, the kiosk
+returns to the main welcome screen.
```
```suggestion
At any point in the check-in process, if ten (10) seconds pass with no selection, the kiosk
returns to the main welcome screen.
```

jero-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+line for the visitor who was served. Once they are marked as having their drink served, the visitor
+disappears from the list.
+
+Visitors do not check themselves out when they leave. Once the visitor completed their business and
```
```suggestion
Visitors do **not** check themselves out when they leave. Once the visitor completed their business and
```
Just a suggestion 😉

jero-odoo reviewed (changes requested):
Hey @larm-odoo great job! Let me know if you have any questions, tag me when you want me to re-review!

larm-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Cancelled`. In the default view, only
+  guests with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible.
+- :guilabel:`Email`: The guest's email address.
+- Status Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
```
```suggestion
- :guilabel:`Status` Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
```

larm-odoo replied:
I don't know why the comment wasn't saved, I know I already explained the reason why I didn't go through with this change- so here it is again. I originally agreed, but then changed my mind. The reason is, there is no text I am quoting on the list. The column header is blank, so that's why Status Column is not capitalized- I am not quoting something on the screen.

larm-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+:menuselection:`Frontdesk application --> Visitors`. The visitors are listed in a list view, with
+the following details that were entered when they checked-in:
+
+- :guilabel:`Name`: The guest's name.
+- :guilabel:`Visitor Company`: The guest's company they represent.
+- :guilabel:`Phone`: The guest's phone number.
+- :guilabel:`Drinks`: The drink the guest requested.
+- :guilabel:`Host`: Who the guest is waiting to see.
+- :guilabel:`CheckIn`: The date and time the guest checked-in.
+- :guilabel:`Checkout`: The date and time the guest checked-out. In the default view, only guests
+  with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible. Guests with check-out
+  times are only visible if the filter :guilabel:`Today` is not active.
+- :guilabel:`Duration`: The current amount of time the guest has been checked-in for.
+- :guilabel:`Station`: The location of where the guest checked-in.
+- :guilabel:`Status`: The status of the guest. The options are :guilabel:`Checked-In`,
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Cancelled`. In the default view, only
+  guests with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible.
+- :guilabel:`Email`: The guest's email address.
+- Status Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
+  record and log the date and time they left. If a scheduled guest arrives, if they did not check-in
+  using the *Frontdesk* application kiosk, they can be checked-in on the visitor log. Click
+  :guilabel:`Check in` to log the date and time they arrived.
+- :guilabel:`Drink Served`: When a guest requests a drink, the :guilabel:`Drink Served` button
+  appears at the end of the visitor line. When a drink has been served, click the :guilabel:`Drink
+  Served` button to indicate the drink has been delivered to the guest.
```
I agree with the status- I was torn since I was adding 'column', but it looks better bolded with a GUI label. A lot of the info in the different sections overlaps, but I wanted to be thorough, so if anyone jumped to only one section they had a question on, all the info was there.

larm-odoo commented:
Thanks @jero-odoo  - I added/implemented all your changes.

jero-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+
+If a guest is expected to arrive at a future date and time, it is possible to enter their
+information into the *Frontdesk* application in advance. To create a planned guest, navigate to
+:menuselection:`Frontdesk application --> Visitors  --> New`. Enter the same information as any
```
```suggestion
:menuselection:`Frontdesk application --> Visitors --> New`. Enter the same information as any
```

jero-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+:guilabel:`Nothing, thanks` button at the bottom of the screen. If a drink selection was made, a
+`Thank you for registering! Your drink is on the way.` message appears.
+
+At any point in the check-in process, if ten (10) seconds pass with no selection, the kiosk
+returns to the main welcome screen.
```
```suggestion
At any point in the check-in process, if ten (10) seconds pass with no selection, the kiosk returns to the main welcome screen.
```
Early line break (100 chracters)

jero-odoo reviewed (approved):
Hey @larm-odoo, just two really small changes (spacing issues). Other than that, this is ready for the next round of reviews

larm-odoo commented on content/applications/hr.rst:
```diff
@@ -12,3 +12,4 @@ Human resources
    hr/payroll
    hr/recruitment
    hr/referrals
+   hr/frontdesk
```
Sounds good!

larm-odoo commented:
Hi @jcs-odoo - I moved the order of the doc in the list as suggested. I just tagged you so you can approve that small change. Thank you!

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Frontdesk
+=========
+
+The Odoo *Frontdesk* application is a way for visitors to check in to a building or location and
```
```suggestion
The Odoo *Frontdesk* application provides a way for visitors to check in to a building or location and
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+The Odoo *Frontdesk* application is a way for visitors to check in to a building or location and
+alert the person they are seeing that they have arrived. Additionally, they can request a
+pre-configured beverage, to be brought to them while they wait. This application is ideal for
+businesses that do not have someone working at a reception desk, or perhaps do not have a lobby or
+waiting area available to guests and visitors.
```
```suggestion
businesses that either do not have someone working at a reception desk, or do not have a lobby or
waiting area available to guests and visitors.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Configuration
+=============
+
+The first item to be configured to use the *Frontdesk* application is the station or stations,
+followed by the drinks that are offered.
```
What do you think about moving this sentence inside the heading below, since it's relevant to Stations? Totally optional, and up to you! For example, it would look like:

Configuration
=========

Stations
---------

The first item to be configured ...

larm-odoo replied:
I was told it was best to try and have some text between sections, always- so I think I'll leave this here since there are just 2 configurations and it explains that you do them both.

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Stations
+--------
+
+A **Station** can be thought of as any location where someone can sign-in and wait for an employee.
```
```suggestion
In Odoo *Frontdesk*, a *Station* can be any location where someone can sign-in and wait for an employee.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+
+A **Station** can be thought of as any location where someone can sign-in and wait for an employee.
+This is typically some form of waiting room, such as a lobby. Each station has a kiosk where
+visitors check in. There is a minimum of one station that must be configured to use the *Frontdesk*
+application, but there is no limit to how many stations that can created and configured.
```
```suggestion
visitors are able to check in.

When setting up the *Frontdesk* application, a minimum of one station **must** be configured, but there is no upper limit to how many stations can be created and configured.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Stations` and click :guilabel:`New`. A blank frontdesk form loads. Enter the following information
+on the form:
+
+- :guilabel:`Frontdesk Name`: enter the name for the specific Frontdesk location. This should be
```
```suggestion
- :guilabel:`Frontdesk Name`: enter the name for the specific frontdesk location. This should be
```
It should be lowercase, right? Just basing this on what I'm seeing below, feel free to ignore if I'm wrong though

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+  short and identifiable, such as `Reception Desk` or `Main Lobby`. This field is required in order
+  to create a station.
+- :guilabel:`Responsibles`: select the person or persons who are alerted when a visitor checks in
+  using at specific frontdesk. Multiple selections can be entered. This field is required in order
```
```suggestion
  using a specific frontdesk. Multiple selections can be entered. This field is required in order
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+  populated, the form can be saved manually. Click the :guilabel:`Save Manually` icon (a cloud with
+  an up arrow inside it) and the :guilabel:`Kiosk URL` field populates. This URL is one way the
+  frontdesk kiosk is accessed. To access the kiosk, click the :guilabel:`Copy` button at the end of
+  the URL, and navigate to that URL in a web browser. This URL opens that specific stations'
```
```suggestion
  the URL, and navigate to that URL in a web browser. This URL opens that specific station's
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+
+- :guilabel:`Host Selection`: if the visitor is attending a meeting, this option allows the visitor
+  to select the meeting host from a presented list, and notify that individual.
+- :guilabel:`Authenticate Guest`: if additional information is required when a guest checks-in,
```
```suggestion
- :guilabel:`Authenticate Guest`: if additional information is required when a guest checks in,
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+  - :guilabel:`Organization`: select whether the guest's organization is :guilabel:`Required`,
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+
```
Should this extra line be here between `Optional` and `Theme`? Or is it all the same list?

larm-odoo replied:
Yes, because there's a weird issue when you need to indent, I noticed you have to add a space before and after a section you want indented, or it's messed up. I remember trying a lot of different ways to get it to work but this seems to do it!

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+  - :guilabel:`Organization`: select whether the guest's organization is :guilabel:`Required`,
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+
+- :guilabel:`Theme`: select the color mode of the kiosk. Chose either :guilabel:`Light` or
```
```suggestion
- :guilabel:`Theme`: select the color mode of the kiosk. Choose either :guilabel:`Light` or
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+   An example of how that might appear in a *Discuss* message is: `Main Lobby Check-In: John Doe
+   (123-555-1234) (Odoo, Inc.) to meet Marc Demo.`
+
+
```
Extra line? Can delete if so

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Drinks
+------
+
+After all the stations are created, the next step is to configure the drinks on offer, if desired.
```
```suggestion
After all the stations are created, the next step is to configure the drinks to offer to visitors, if desired.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+configured if drinks are offered to guests.
+
+To add a drink option, navigate to :menuselection:`Frontdesk application --> Configuration -->
+Drinks`. Click :guilabel:`New` and a blank drink form loads. Enter the following information on the
```
```suggestion
Drinks`, and click :guilabel:`New`. This causes a blank drink form to load. Enter the following information on the
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+   :align: center
+   :alt: Drink form with the information filled out for an espresso.
+
+Stations
```
Do you think since there's already another heading called `Stations` near the top of this doc, that this could (or should) be called something else? Just a suggestion, but if it works better this way feel free to ignore

larm-odoo replied:
I can call this section the Station dashboard- this is a bit more descriptive.

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+  guests with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible.
+- :guilabel:`Email`: the guest's email address.
+- Status Column: when a guest leaves, click the :guilabel:`Check out` button to update the guest's
+  record and log the date and time they left. If a scheduled guest arrives, if they did not check-in
```
```suggestion
  record and log the date and time they left. If a scheduled guest arrives, and they did not check in
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+
+   If a guest does check-in using a kiosk, ensure all records are current, and the list of
+   currently on-site guests is correct. Be sure to check in and/or check-out the correct entries so
+   the visitor lists correctly reflects who is currently on the premises.
```
```suggestion
   the visitors list correctly reflects who is currently on the premises.
```
Right?

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+
+.. important::
+   It is recommended to log out of the database and close the tab after navigating to the kiosk, so
+   there is no possibility of a visitor accessing the database when checking-in.
```
```suggestion
   there is no possibility of a visitor accessing the database when checking in.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+in order to check in. Once all the information is entered, the visitor taps the :guilabel:`Check In`
+button.
+
+If drinks were configured for the station, after tapping :guilabel:`Check In`, a registration
+confirmation screen loads, along with the question `Do you want something to drink?`. The visitor
+can tap either :guilabel:`Yes, please`, or :guilabel:`No, thank you`. If they select :guilabel:`Yes,
+please`, a drink selection screen appears, and the pre-configured options are listed. The visitor
+then taps the desired selection, or if they do not want anything, they can tap the
+:guilabel:`Nothing, thanks` button at the bottom of the screen. If a drink selection was made, a
+`Thank you for registering! Your drink is on the way.` message appears.
```
Just a general question - is there any way to change these default responses to selections on the beverage screen? Or are we stuck with them, and can't change them if desired?

larm-odoo replied:
There's no way to change them that I can tell- and if it is a deep in the depths of Odoo modification, I didn't want to include it. From what I can tell, this is standard and I can't change these with anything in the application.

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+line for the visitor who was served. Once they are marked as having their drink served, the visitor
+disappears from the list.
+
+Visitors do **not** check themselves out when they leave. Once the visitor completed their business
+and has left the premises, it is important to check them out for accurate record keeping. Navigate
```
```suggestion
Visitors do **not** check themselves out when they leave. Once the visitor has completed their business
and left the premises, it is important to check them out for accurate record keeping. Navigate
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+disappears from the list.
+
+.. important::
+   Always have an accurate list of who is on the premises at any given time. Since visitors do not
+   check themselves out, it is important for accurate record keeping to check-out visitors when they
+   leave. This is important both in the case of an emergency, and for security purposes.
```
```suggestion
   Always have an accurate list of who is on the premises at any given time. Since visitors do not
   check themselves out, it is important to check out visitors when they leave, for accurate record keeping. This is 
   important both in the case of an emergency, and for security purposes.
```

brse-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,318 @@
+Reporting
+=========
+
+The *Frontdesk* application has two reports available, :guilabel:`Visitors` and :guilabel:`Drinks`.
```
```suggestion
The *Frontdesk* application has two reports available: :guilabel:`Visitors` and :guilabel:`Drinks`.
```

brse-odoo reviewed (approved):
Hi @larm-odoo, I've just finished my review of this doc. Great job! It was a lot of information, but very nicely segmented, straightforward, and easy to digest. I left some comments/suggestions for minor grammatical stuff, and some general inquiries.

Feel free to reach out if you have any questions, thank you!

ksc-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,325 @@
+At any point in the check-in process, if ten (10) seconds pass with no selection, the kiosk returns
+to the main welcome screen.
+
+Once the visitor has checked in, both the person they are visiting, and any users who were
+configured to be notified when anyone checks-in at the kiosk are notified. The notification is
+either by email, SMS message, a *Discuss* chat, or any combination of those three options.
```
```suggestion
Once the visitor has checked-in, the person they are visiting, and any other users who were
configured to be notified when checks-in occur at the kiosk, are notified. The notification is
either by email, SMS message, a *Discuss* chat, or any combination of those three options.
```

ksc-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,325 @@
+configured to be notified when anyone checks-in at the kiosk are notified. The notification is
+either by email, SMS message, a *Discuss* chat, or any combination of those three options.
+
+If the visitor requested a drink, the user or users who were configured as the :guilabel:`People to
+Notify` on the drink form, are notified via the *Discuss* application. The message that appears is:
+`(Visitor Name) just checked-in. They requested (drink).`
```
```suggestion
If the visitor requested a drink, the user(s) configured as the :guilabel:`People to Notify` on the
drink form, are notified via the *Discuss* application. The message that appears is: `(Visitor Name)
just checked-in. They requested (Drink Name).`
```

ksc-odoo commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,325 @@
+.. important::
+   Always have an accurate list of who is on the premises at any given time. Since visitors do not
+   check themselves out, it is important to check out visitors when they leave, for accurate record
+   keeping. This is important both in the case of an emergency, and for security purposes.
```
```suggestion
   keeping.
   
   This is important for security purposes, and in the case of an emergency.
```

ksc-odoo reviewed (approved):
Hey @larm-odoo -- Just finished my Final Review of this doc. Awesome job! Very in-depth 👍 I've left a decent amount of feedback that'll need your attention, but once you address those issues, and make the necessary adjustments, I think this one will be ready for Tech Review. Also, it might be wise to ensure that the images you've screenshot are clear. Currently, they appear a bit blurry, and require a decent amount of "squinting" to make out what's being shown. So, be sure to give those a look, as well. But, once again, fantastic job with this Frontdesk doc, and if you have any questions, or need clarification on anything, don't hesitate to reach out. 👍

larm-odoo commented:
Thanks for the thorough review, @ksc-odoo! I made all the changes requested and I retook all the images. One is still blurrier than I'd like, for the large image that needs to be zoomed out enough to capture it all in the screenshot. The others are more crisp now! I will tag tech review now- thank you!

ksc-odoo reviewed (approved):
Hey there! @larm-odoo ... Thanks for making all the necessary changes. Since I'm part of the Final review team (us-doc-review), you can feel free to remove the request for me to review this again, since I gave it a very thorough review yesterday and you've let me know all the modifications have been made. So, feel free to just request a review from @samueljlieber for Tech Review. Then, once it's through that stage, you can tag ZST for the very last step. Thanks! If you have any questions, just let me know. 👍

samueljlieber commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,405 @@
+and alert the person they are meeting about their arrival. Additionally, they can request a
+pre-configured beverage to be brought to them, while they wait.
+
+This application is ideal for businesses that do **not** have someone working at a reception
+desk, locations **without** a designated waiting area available to guests and visitors.
```
Early line break
```suggestion
This application is ideal for businesses that do **not** have someone working at a reception desk,
locations **without** a designated waiting area available to guests and visitors.
```

samueljlieber commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,405 @@
+  To access the kiosk, click the :guilabel:`Copy` button at the end of the URL, and navigate to that
+  URL in a web browser. This URL opens that specific station's frontdesk sign-in page.
+
+  .. tip::
+   To add an image/photo to a frontdesk form, hover over the :guilabel:`(camera with a '+' sign)`
+   icon in the top-right of the form to reveal a :guilabel:`✏️ (pencil)` icon.
+
+   Click on the :guilabel:`✏️ (pencil)` icon to open a file explorer, navigate to the desired
+   image/photo file, then click :guilabel:`Open` to select it.
+
+   The image selected for the station photo appears as the background image for the station kiosk.
```
Indent the text in this admonition please
```suggestion
  .. tip::
     To add an image/photo to a frontdesk form, hover over the :guilabel:`(camera with a '+' sign)`
     icon in the top-right of the form to reveal a :guilabel:`✏️ (pencil)` icon.

     Click on the :guilabel:`✏️ (pencil)` icon to open a file explorer, navigate to the desired
     image/photo file, then click :guilabel:`Open` to select it.

     The image selected for the station photo appears as the background image for the station kiosk.
```

samueljlieber commented on content/applications/hr/frontdesk.rst:
```diff
@@ -0,0 +1,405 @@
+  end of the line, and make any desired edits to the content of the template. The SMS message may
+  have a maximum of 242 characters, which fits in 4 SMS (UNICODE) messages.
+- :guilabel:`Notify by Discuss`: this option is enabled by default when the :guilabel:`Host
+  Selection` option is enabled. This option opens a *Discuss* application message window
+  with the person the guest is visiting upon check-in.
```
Early line break
```suggestion
  Selection` option is enabled. This option opens a *Discuss* application message window with the
  person the guest is visiting upon check-in.
```

samueljlieber commented on content/applications/hr/frontdesk/station-form.png:
This image is pretty difficult to see due to the size of the text and showing the entire UI. 

If you are going to keep this image, I'd like to see it retaken with more zoom and resizing the browser window so there isnt so much empty space on the right (you can even crop out the Photo field).

I'd even suggest removing this image entirely, since your description of the the Frontdesk form is very thorough 🙂

larm-odoo replied:
I changed this to crop out the image- I did that originally but then was worried it didn't include it. So I have retaken it, and it is better lookin gnow. If you don't like how it looks still though, I can remove it- just figured I'd try to keep it first.

samueljlieber replied:
Its much better 🙂 thank you!

samueljlieber reviewed (changes requested):
Hi @larm-odoo, nice work on this PR! Its exciting to see the HR docs coming together! 

I only have a few corrections and suggestions, please see below 🙂

Also, its a nitpick, the commit name and PR title could be a little more concise, something like: 
```
[ADD] Frontdesk: add new configuration doc
```

larm-odoo commented:
Hi @samueljlieber - I updated the pic and made the requested edits. I updated the name for the PR but wasn't 100% sure how to change the commit message without messing it up, so I left the commit name as it is.  Ready for you again!

samueljlieber reviewed (approved):
Looks good to me @larm-odoo! Nice job, the image is much better thank you!

I pushed up a commit https://github.com/odoo/documentation/commit/0b161dcd52beaef391b0d41dae66f4cbe74ace11 to update the commit message (using `git commit --amend`).

samueljlieber commented:
@StraubCreative this PR is good to go 👍

StraubCreative reviewed (changes requested):
Hi @larm-odoo 
I'm catching a number of grammar issues as I work my way through the doc.
Can you take another look please?
Hint: consider using an AI tool to proofread your doc for you 😉

larm-odoo commented:
Hi @StraubCreative - I rechecked my doc, and I used both google doc and chatgpt to proofread this. I did try chatgpt last week, but it gave really odd results, so I had switched to google (it found more spelling than grammar, but was still a good tool). I tried both again- and today's try with chatgpt was better, BUT, it did say the following for almost every section I pasted in to check:
"I didn't detect any significant grammar issues in the text snippets you provided today. The content appears to be well-structured and grammatically correct. If you have more text or specific sections you'd like me to review, feel free to share them!"
So in the end I mostly relied on just reading it myself, since I got very few actual edits from them. I did redo all instances of 'check-in' vs 'check in' according to our new doc, and those were the bulk of my edits along with consolidating a couple of important blocks, and a few wording changes here and there.

StraubCreative reviewed (approved):
Seems good! Thanks @larm-odoo 🚀 

@robodoo r+
